src/daisy.js

```diff
@@ -132,12 +149,60 @@ var daisy = (function(Object, Array, Math, undefined){
 	var Computation = function(initParam, options){
 		this.currentVal = initParam + '';
 		this.options = options || {};
+		if (this.options.round && !this.options.format) {
+			this.options.format = 'n';
+		}
+		if (this.options.INR && this.options.commas === void 0) {
+			this.options.commas = true;
+		}
+	};
+
+	Computation.prototype.format = function () {
+		var formatted,

```

Comment:
I think is is the biggest function we have here.  Can we break it up a bit?

I'm a big fan of nested functions... something like

```javascript
var bigFunction = function(){
    // define helper functions inside
    var smallerFunction = function(){
         // stuff...
    };

    var evenSmaller = function(){
        // more stuff...
    };

    // save invocations for the end
    smallerFunction():
    evenSmaller();
}
```